src/pages/index.js

```diff
@@ -1,106 +1,204 @@
   cardListEl,
   previewImageModal,
-} from '../utils/constants.js';
+  initialCards, 
+  settings
+} from '../utils/constants';
+import Api from '../utils/Api';
 
 /* -------------------------------------------------------------------------- */
-/*                               Form Validator                               */
+/*                                  Instances                                 */
 /* -------------------------------------------------------------------------- */
 
-const editFormValidator = new FormValidator(validationSettings, profileEditModal);
-const addFormValidator = new FormValidator(validationSettings, profileAddModal);
-editFormValidator.enableValidation();
-addFormValidator.enableValidation();
+const editProfileFormValidator = new FormValidator(settings, profileEditModal);
+const addCardFormValidator = new FormValidator(settings, cardModal);
+const avatarFromValidator = new FormValidator(settings, profileAvatarModal);
+const deleteCardFormValidator = new FormValidator(settings, deleteCardModal);
+const userInfo = new UserInfo(profileTitle, profileDescription, profileAvatar);
+const editProfilePopup = new PopupWithForm(profileEditModal, handleProfileFormSubmit);
+const previewImagePopup = new PopupWithImage(previewImageModal);
+const addNewCardPopup = new PopupWithForm(cardModal, handleSubmitCard);
+const deleteImagePopup = new PopupWithConfirm(deleteCardModal);
+const editAvatarPopup = new PopupWithForm(profileAvatarModal, handleProfileAvatarSubmit);
 
 /* -------------------------------------------------------------------------- */
-/*                                   Profile                                  */
+/*                               Form Validators                              */
 /* -------------------------------------------------------------------------- */
 
-const userInfo = new UserInfo({
-  nameSelector: '.profile__title', 
-  jobSelector: '.profile__description'
-});
-
-const editProfilePopup = new PopupWithForm(profileEditModal, ({ title, description }) => {
-  userInfo.setUserInfo({ title, job: description });
-  editProfilePopup.close();
-});
-
-profileEditBtn.addEventListener('click', openProfilePopup);
+editProfileFormValidator.enableValidation();
+addCardFormValidator.enableValidation();
+avatarFromValidator.enableValidation();
+deleteCardFormValidator.enableValidation();
 
-function openProfilePopup() {
-  const { name, job } = userInfo.getUserInfo();
-  profileTitleInput.value = name;
-  profileDescriptionInput.value = job;
-  editFormValidator.toggleButtonState();
-  editProfilePopup.open();
-}
+/* -------------------------------------------------------------------------- */
+/*                                 Api Promise                                */
+/* -------------------------------------------------------------------------- */
 
-editProfilePopup.setEventListeners();
+let userId;
+let cardList;
+
+Promise.all([Api.getInitialCards(), Api.getUserInfo()])
```

Comment:
The methods should not be static, you should use api class instance `api` methods